Bak File/Read.me

```diff
@@ -0,0 +1 @@
+We have included the Bak file to make it easy for you to restore on your own computer.
```

Comment:
Again, you're not putting these bak files or tables into a carpet cleaning folder.  That's not good.  You're repo is going to contain more projects than just this carpet cleaning database project.  So we need to have those boundaries clearly defined.  Please put these bak files and tables one more table deep.  CarpetCleaning/BakFiles/CleanGreenStl.bak.  Also I thought we were gonna take the name Clean Green out of all of this?

Reply:
Moved them into the Carpet cleaning folder. Geez.